backend/app.js

```diff
@@ -29,25 +29,25 @@ app.use('/agency', require('./routes/agency'));
 
 // Catch-all route
 app.get('/*', (req, res, next) => {
-    res.sendFile(path.join(__dirname, 'build/index.html'), (err) => {
-        if (err) {
-            next();
-        }
-    });
+  res.sendFile(path.join(__dirname, 'build/index.html'), (err) => {
+	if (err) {
+	  next();
+	}
+  });
```

Comment:
Lines 33 - 35 does not conform to 2 spacing.

Reply:
should be fixed!!